app/views/api/v2/hosts/main.json.rabl

```diff
@@ -40,6 +40,18 @@ node :hostgroup_title do |host|
   host.hostgroup.title if host.hostgroup.present?
 end
 
+if params[:action] == 'show' || (params[:include].present? && params[:include].include?('parameters'))
```

Comment:
Can you extract `params` out of the view? It ties the view to this controller quite tightly and may prevent foreman_hooks from using the view via `Rabl.render`. It might be better to use an instance variable set in the controller(s) as these can be passed into `Rabl.render`.